### PR TITLE
Modified demos to add new controller support

### PR DESCRIPTION
**Pull request by NolaDonato**
Modified demos to support Gear controller by using new controller API GVRInputManager.selectController. Heavily modified 3D cursor demos to actually use the features in the 3D cursor library instead of duplicating them.

This PR require GearVRf PR #1628.

Each demo is a separate commit. The demos changed are: 
gvr-3dcursor
gvr-3dcursor-simple
gvr-keyboardview
gvr-simplephysics
gvr-controls
gvr-immersivepedia
gvr-events
gvr-controller
gvr-accessibility

### CODE REVIEW

File: gvr-accessibility/app/src/main/assets/gvr.xml [hunk 1]
```diff
@@ -28,9 +28,10 @@
         framebufferPixelsHigh="DEFAULT"
         framebufferPixelsWide="DEFAULT"
         showLoadingIcon="true"
-        useMultiview="true"
+        useMultiview="false"
```
Comment (liaxim) [inline gvr-accessibility/app/src/main/assets/gvr.xml]: Why multiview needs to be turned off?

Comment (NolaDonato) [reply gvr-accessibility/app/src/main/assets/gvr.xml]: It doesn't. I will turn it back on. I was testing both cases.

**Comment by liaxim** [comment]
gvr-accessibility: enable all options on the circular menu near the bottom; for me when I click the Zoom out menu, the app registers Volume down action. Most if not options from this menu do not do what you would expect. Tested on S8.

**Comment by liaxim** [comment]
gvr-simplephysics: not sure it is playable with the controller the way it is or maybe it is just me.

**Comment by liaxim** [comment]
gvr-keyboardview: keyboard view clearly shows the picking problem apparent in most other apps

**Comment by liaxim** [comment]
gvr-3dcursor: it just doesn't do what it used to do; can't select anything with a touch event. All my tests btw were in developer mode without headset.

**Comment by liaxim** [comment]
gvr-3dcursor-simple: the controller is there unlike gvr-3dcursor; not quite sure you can do the same things you could in the past but have not tested with the version without the changes.

**Pull request merged**